languages/javascript/reference/info/scope.md

```diff
@@ -0,0 +1,7 @@
+## Scope / Visibility
+
+Most languages have some concept of encapsulation through scope and visibility.
+Tracks should explain the language-concepts and keywords that their track uses (e.g. `public`, `private`, `export`) and what the normal way of using those scopes are in their language. An exercise may be appropriate, but providing examples in stubs early on may avoid overwhelming the student.
+
+[Global Scope](#global-scope)
+https://developer.mozilla.org/en-US/docs/Glossary/Global_scope
```

Comment:
I think I would rather write a (probably short) sentence here about global scope, and then add a link to the mozilla docs (if that is even necessary, since there are just two sentences on it).